src/mir/index.md

```diff
@@ -44,7 +44,8 @@ This section introduces the key concepts of MIR, summarized here:
   leading underscore, like `_1`. There is also a special "local"
   (`_0`) allocated to store the return value.
 - **Places:** expressions that identify a location in memory, like `_1` or
-  `_1.f`.
+  `_1.f`. (Some people call these "lvalues"; the "l" meaning that they occur on
+  the left-hand side of an assignment.)
 - **Rvalues:** expressions that produce a value. The "R" stands for
```

Comment:
Similarly this should be 'value expressions'.
https://doc.rust-lang.org/reference/expressions.html#place-expressions-and-value-expressions

Reply:
Actually it seems they're called ["rvalues" in MIR](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/enum.Rvalue.html).